### PR TITLE
[MRG] Sphinx, narrative documentation...

### PR DESCRIPTION
**Pull request by thomasbazeille**
Also includes improved docstrings, more complete package informations, cleaner examples

### CODE REVIEW

File: fmralign/template_alignment.py [hunk 1]
```diff
@@ -295,13 +293,12 @@ def __init__(self, alignment_method="identity", n_pieces=1,
...
         """
-        Learn a template from images, using alignment.
+        Learn a template from source images, using alignment.
 
         Parameters
         ----------
         imgs: List of Niimg-like objects
-           See http://nilearn.github.io/manipulating_images/input_output.html
-           source data. Every img must have the same length (number of sample)
+            Source subjects data. Every img must have the same length (number of sample).
```
Comment (bthirion) [inline fmralign/template_alignment.py]:
```suggestion
            Source subjects data. Every img must have the same length (number of samples).
```

File: source/functional_alignment/functional_alignment_intro.rst [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+even in simple and narrow experimental settings.
+
+Functional alignment
+=====================
+
+The core idea of functional alignment is to handle this variablity by learning a transformation
+from one subject to another.
+
+First, we choose one **target** subject, toward which we'll transform images from a **source** subject. We use shared information to learn the transformation from source to target. We won't get now into
+details about how the transformation is learnt. More details on the possibility
```
Comment (bthirion) [inline source/functional_alignment/functional_alignment_intro.rst]:
```suggestion
details about how the transformation is learnt. More details on the features
```

**Comment by bthirion** [comment]
OK, the output is good. Minor improvements are possible, but the current version is good enough.
Maybe simply fix the minro comments i made, and we merge.
Thx !

File: AUTHORS.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+--------
+
+This project has received funding from the European Union’s Horizon
+2020 Research and Innovation Programme under Grant Agreement No. 785907
+(HBP SGA2).
+fmralign is also supported by `Digiteo <https://digiteo.fr>`_
+
+.. _citing:
+
+Citing scikit-learn
```
[the remark this answers is not in the document]
Comment (thomasbazeille) [reply AUTHORS.rst]: I think it's a comment that doesn't concern fmralign only. I just replicated what is written on nilearn : "there's no paper released on nilearn, please cite scikit-learn for the time being"

[the remark this answers is not in the document]
Comment (bthirion) [reply AUTHORS.rst]: Sorry, to be more precise, people should cite :
https://hal.inria.fr/hal-01093971
Best,
B

**Pull request merged**